DirectEmployers/MyJobs
# [PD-3351] refactor analytics css

## What's this PR do?
PR restructures the css for analytics

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of any of the analytics apps

## How should this be manually tested?
Pull down repository. Check the css files and makes sure the analytics app is the same

## Any background context you want to provide?
Reopened to edit JS files that were breaking
